DependencyInjection/TacticianExtension.php

```diff
@@ -71,4 +73,25 @@ private function injectMethodNameInflector(array $mergedConfig, ContainerBuilder
         $handlerLocator = $container->findDefinition('tactician.middleware.command_handler');
         $handlerLocator->replaceArgument(2, $inflectorReference);
     }
+
+    /**
+     * Configure the security voter.
+     *
+     * @param array $mergedConfig
+     * @param ContainerBuilder $container
+     */
+    private function configureSecurity(array $mergedConfig, ContainerBuilder $container)
+    {
+        if (!$container->has('tactician.middleware.security_voter')) {
+            $definition = new Definition(
+                HandleCommandVoter::class,
+                [
+                    new Reference('security.access.decision_manager'),
+                    $mergedConfig['security']
```

Comment:
Good point, I don't really know if it's required. Still, a check is cheap and safe, so I don't see a downside to adding it regardless.